stock_available/views/res_config_view.xml

```diff
@@ -22,6 +22,8 @@
                                 <div>
                                     <field name="module_stock_available_mrp" class="oe_inline" />
                                     <label for="module_stock_available_mrp" />
+                                    <label for="stock_available_mrp_based_on" />
+                                    <field name="stock_available_mrp_based_on" class="oe_inline" attrs="{'required':[('module_stock_available_mrp','=',True)]}"/>
```

Comment:
Is there a reason not to put this new setting directly in an inherit model in stock_available_mrp (so this PR changes only stock_available_mrp)?

(Sorry if this question is strange, I'm a new Odoo developer :-))